sysl2/sysl/seqs.go

```diff
@@ -147,34 +135,36 @@ func DoGenerateSequenceDiagrams(stdout, stderr io.Writer, flags *flag.FlagSet, a
+	sd := kingpin.New("sd", "Generate sequence diagram")
+	root := sd.Flag("root", "sysl root directory for input model file (default: .)").Default(".").String()
+	endpoint_format := sd.Flag("endpoint_format", "Specify the format string for sequence diagram endpoints. "+
+		"May include %%(epname), %%(eplongname) and %%(@foo) for attribute foo(default: %(epname))").Default("%(epname)").String()
+	app_format := sd.Flag("app_format", "Specify the format string for sequence diagram participants. "+
+		"May include %%(appname) and %%(@foo) for attribute foo(default: %(appname))").Default("%(appname)").String()
+	title := sd.Flag("title", "diagram title").Short('t').String()
+	plantuml := sd.Flag("plantuml", strings.Join([]string{"base url of plantuml server",
 		"(default: $SYSL_PLANTUML or http://localhost:8080/plantuml",
-		"see http://plantuml.com/server.html#install for more info)"}, "\n"))
-	output := flags.String("output", "%(epname).png", "output file(default: %(epname).png)")
-	modules_flag := flags.String("modules", ".", strings.Join([]string{"input files without .sysl extension and with leading /",
-		"eg: /project_dir/my_models",
-		"combine with --root if needed"}, "\n"))
-	flags.Var(&endpoints_flag, "endpoint", "Include endpoint in sequence diagram")
-	flags.Var(&apps_flag, "app", "Include all endpoints for app in sequence diagram (currently "+
+		"see http://plantuml.com/server.html#install for more info)"}, "\n")).Short('p').String()
+	output := sd.Flag("output", "output file(default: %(epname).png)").Default("%(epname).png").Short('o').String()
+	endpoints_flag := sd.Flag("endpoint", "Include endpoint in sequence diagram").Short('s').Strings()
+	apps_flag := sd.Flag("app", "Include all endpoints for app in sequence diagram (currently "+
 		"only works with templated --output). Use SYSL_SD_FILTERS env (a "+
-		"comma-list of shell globs) to limit the diagrams generated")
-	flags.Var(&blackboxes_flag, "blackbox", "Apps to be treated as black boxes")
+		"comma-list of shell globs) to limit the diagrams generated").Short('a').Strings()
+	blackboxes_flag := sd.Flag("blackbox", "Apps to be treated as black boxes").Strings()
 
 	// Following variables currently are unused. Keep them to align with the python version.
-	filter := flags.String("filter", "", "Only generate diagrams whose output paths match a pattern")
-	no_activations := flags.Bool("no-activations", true, "Suppress sequence diagram activation bars(default: true)")
-	verbose := flags.Bool("verbose", false, "Report each output(default: false)")
-	expire_cache := flags.Bool("expire-cache", false, "Expire cache entries to force checking against real destination(default: false)")
-	dry_run := flags.Bool("dry-run", false, "Don't perform confluence uploads, but show what would have happened(default: false)")
+	filter := sd.Flag("filter", "Only generate diagrams whose output paths match a pattern").String()
```

Comment:
remove these if we are not handling them.

Reply:
removed.